2019-09-21 17:48:15,312 CRITICAL server_log_config.py Test critical event
2019-09-21 17:48:15,328 ERROR server_log_config.py Test error event
2019-09-21 17:48:15,328 DEBUG server_log_config.py Test debug event
2019-09-21 17:48:15,328 INFO server_log_config.py Test info event
2019-09-21 17:55:53,582 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2019-09-21 17:55:58,411 INFO server.py Установлено соедение с ПК ('127.0.0.1', 53195)
2019-09-21 17:55:58,411 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1569077758.411991, 'user': {'account_name': 'User'}}
2019-09-21 17:55:58,411 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1569077758.411991, 'user': {'account_name': 'User'}}
2019-09-21 17:55:58,411 INFO server.py Cформирован ответ клиенту {'response': 200}
2019-09-21 17:55:58,411 DEBUG server.py Соединение с клиентом ('127.0.0.1', 53195) закрывается.
2019-09-21 18:02:25,971 INFO server.py Установлено соедение с ПК ('127.0.0.1', 53208)
2019-09-21 18:02:25,972 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1569078145.9719822, 'user': {'account_name': 'User'}}
2019-09-21 18:02:25,972 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1569078145.9719822, 'user': {'account_name': 'User'}}
2019-09-21 18:02:25,972 INFO server.py Cформирован ответ клиенту {'response': 200}
2019-09-21 18:02:25,973 DEBUG server.py Соединение с клиентом ('127.0.0.1', 53208) закрывается.
